src/site/includes/header.html

```diff
@@ -86,6 +86,15 @@
 </head>
 
 <body class="{{ body_class }} merger">
+  <!-- Draft status -->
+  {% if !entityPublished %}
+    <div class="vads-u-background-color--primary-alt-lightest vads-u-padding--1">
+      <div class="vads-l-grid-container medium-screen:vads-u-padding-x--0">
+        You are viewing a draft of "{{ entityUrl.path }}". <a data-same-tab href="{{ drupalSite }}/node/{{ entityId }}/edit">Edit this page in the CMS.</a>
+      </div>
+    </div>
+  {% endif %}
```

Comment:
Love this. We maybe want to include `isPreview`.

```
{% if !entityPublished or isPreview %}

```

Or The `isPreview` just checks for the preview server and would make that banner show even if the page isn't a draft but is still being viewed as a preview

Reply:
Using `&&` for isPreview, seemed to work as desired locally and that way it doesn't show up on random pages 👍

Reply:
Is `&&` correct? I thought Liquid used the word `and` lol

Reply:
I think I've seen both used so I wasn't sure which one to use, do you know by any chance?